android/src/main/cpp/NativeProxy.cpp

```diff
@@ -357,10 +358,26 @@ void NativeProxy::installJSIBindings(
         jsi::Object yogaValues(rt);
         for (const auto &entry : *values) {
           try {
-            auto key =
-                jsi::String::createFromAscii(rt, entry.first->toStdString());
-            auto value = stod(entry.second->toStdString());
-            yogaValues.setProperty(rt, key, value);
+            std::string keyString = entry.first->toStdString();
+            std::string valueString = entry.second->toStdString();
```

Comment:
Why do we call `toStdString()` and then parse the strings here? Can't we access the values more directly?

Reply:
I don't like the current solution, but this was the easiest way to achieve it without changing a lot of codebase from layout animation. I think it should be done in another PR in the future because I don't want to increase the complexity of this PR.